src/main/java/com/exasol/ReleaseRobot.java

```diff
@@ -2,20 +2,23 @@
 
 import static com.exasol.Platform.PlatformName.GITHUB;
 
-import java.util.HashSet;
```

Comment:
I would suggest to create an own package for release robot and move all your packages in that sub-package. By that you avoid collisions with other exasol repositories.

Reply:
Good point!